commons-extension-webapp/src/main/webapp/ckeditorCustom/contents.less

```diff
@@ -0,0 +1,291 @@
+/*
+Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+@import (css) "/rest/v1/platform/branding/css";
```

Comment:
Not needed since this has to be added by CKEditor config, see it here :
https://github.com/Meeds-io/commons/blob/9606e13b93605a4fd94c1f153d51f19d22a72d02/commons-extension-webapp/src/main/webapp/ckeditorCustom/config.js#L71-L73

![image](https://github.com/Meeds-io/commons/assets/594824/d053f199-a6d3-469f-a8f4-258b2861981e)

Reply:
You can add this in CKEditor the same way to ensure using the same style of Portal inside the CKEditor to ensure a WYSIWYG effect